admin_guide/scheduling/descheduler.adoc

```diff
@@ -0,0 +1,326 @@
+[[admin-guide-descheduler]]
+= Descheduling
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Descheculing involves evicting pods based on xref:admin-guide-descheduler-policies[specific policies] so that the pods can be rescheduled
+onto more appropriate nodes. 
+
+Your cluster can benefit from descheduling and rescheduling already-running pods for various reasons:
+
+* A nodes is under- or over-utilized.
+* Pod and node affinity requirements, such as taints or labels, have changed and the original scheduling decisions are longer appropriate for certain nodes.
+* Node failure requires pods to be moved.
+* New nodes are added to clusters.
+
+The descheduler does not schedule replacement of evicted pods. The xref:../../admin_guide/scheduling/scheduler.adoc#admin-guide-scheduler[scheduler]
+automatically performs this task for the evicted pods.
+
+[NOTE]
+====
+The descheduler job is considered a _critical pod_, which prevents the descheduler pod from being evicted by the descheduler. 
+You can make a pod a critical pod by adding the the `scheduler.alpha.kubernetes.io/critical-pod` annotation to the pod specification 
```

Comment:
I'm not sure this is the place to describe how to make a pod critical, especially if there's no context to why a pod should be critical. I'd suggest removing this, or putting in a little more context.